toonz/sources/toonz/sceneviewer.cpp

```diff
@@ -1019,8 +1019,10 @@ void SceneViewer::drawBackground()
 		PROC proc = wglGetProcAddress("glCheckFramebufferStatusEXT");
 		if (proc != nullptr)
 			status = reinterpret_cast<PFNGLCHECKFRAMEBUFFERSTATUSEXTPROC>(proc)(GL_FRAMEBUFFER);
-#else
+#elif MACOSX
 		status = glCheckFramebufferStatus(GL_FRAMEBUFFER);
+#else
+		status = 0;  /* XXX, stub */
```

Comment:
This should be able to be made working, the issue is that (on ArchLinux at least), `gl.h` doesn't define `glCheckFramebufferStatus`. I needed to include `glew.h`, which was conflicting with Qt's OpenGL includes. (did some quick checks that it wasn't caused by order), and seems not.
So likely theres a solution here. but in a few min of testing I couldn't find one (hence the stub).